truss/templates/control/proxy.conf

```diff
@@ -0,0 +1,28 @@
+
+server {
+    listen 8080;
+
+    # Liveness
+    location / {
+        add_header Content-Type text/plain;
+        return 200 "";
+    }
+
+    # Readiness
+    location ~ ^/v1/ {
+        if (!-f "~/inference_server_crashed.txt") {
+            return 503 '{"error": "Model load failed"}';
+        }
+
+        proxy_redirect off;
```

Comment:
worth testing streaming here, I  think you might have to add:

```
proxy_buffering off;
```
to this `location` directive. happy to take a stab at it

Reply:
sure. We didn't add that for the TGI one and it streamed fine. Happy to add it if you think that's best.

Reply:
oh ok awesome, if you've tested it and it works let's not include it